src/driver/argument_parser.hpp

```diff
@@ -151,7 +151,15 @@ struct value_parser
         T result;
         std::stringstream ss;
         ss.str(x);
-        ss >> result;
+        // handle whitespace in string
+        if constexpr(std::is_same<T, std::string>::value)
+        {
+            result = ss.str();
```

Comment:
There is no point in constructing a stringstream if we are just returning a string. Just do `return x`.